app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,51 @@ def update
         end
       end
 
+      def exist
+        userData = params[:user]
+        Rails.logger.info "************ exist is = #{userData[:email]}"
+        user = User.where(email: userData[:email]).first
+        Rails.logger.info "************ user is = #{user}"
+        @redirect_path = '/profile' #change this to actually redirect to the social login function
+        unless user
+             @redirect_path = '/additional_info'
+        end
+        Rails.logger.info "!!!!!!!!!!!! path is #{@redirect_path}"
+        render json: {
```

Comment:
Small nit.  Since we are only returning one `key: value` pair, perhaps reduce this down to one line.  i.e.

```ruby
render json: { redirect_to: @redirect_path }
```

Reply:
done.